app/assets/javascripts/components/buttons/generic-button-component.js

```diff
@@ -0,0 +1,41 @@
+ManageIQ.angular.app.component('genericButtonComponent', {
+  bindings: {
+    angularForm: '=',
+    modelCopy: '=',
```

Comment:
- `angularForm`
Yes
- `saveable`
Probably not needed, since Add/Save are not implemented here.
- `onSave`
Specified only in the bindings, but the implementation should be in the main component
- `onCancel` 
Yes
- `onReset`
Yes
(but, didn't you say above, you didn't want Reset here?)
- `newRecord`
Yes

> Handling redirection and flash messages is not the job for a buttons component.

Don't want to do it in the main component either.
So where should it be?